README.md

```diff
@@ -1,2 +1,5 @@
 # hello-world
-Discovering GitHub through its 'Hello World project'
+Discovering GitHub through its **Hello World project**
```

Comment:
This makes it more read-able. _Italic_ would have worked better in **my** opinion ...